src/WebJobs.Script/Diagnostics/Extensions/LoggerExtension.cs

```diff
@@ -12,190 +12,164 @@ internal static class LoggerExtension
+           LoggerMessage.Define<string, Guid>(LogLevel.Debug,
+               new EventId(329, nameof(CustomHandlerSendingInvocation)),
+               "Sending invocation for function: '{functionName}' invocationId: '{invocationId}'");
 
         private static readonly Action<ILogger, string, Guid, Exception> _customHandlerReceivedInvocationResponse =
-           LoggerMessage.Define<string, Guid>(
-           LogLevel.Debug,
-           new EventId(330, nameof(CustomHandlerReceivedInvocationResponse)),
-           "Received invocation response for function: '{functionName}' invocationId: '{invocationId}'");
+           LoggerMessage.Define<string, Guid>(LogLevel.Debug,
+               new EventId(330, nameof(CustomHandlerReceivedInvocationResponse)),
+               "Received invocation response for function: '{functionName}' invocationId: '{invocationId}'");
 
         private static readonly Action<ILogger, Exception> _jobHostFunctionTimeoutNotSet =
-            LoggerMessage.Define(
-            LogLevel.Information,
-            new EventId(331, nameof(JobHostFunctionTimeoutNotSet)),
-            "FunctionTimeout is not set.");
+            LoggerMessage.Define(LogLevel.Information,
+                new EventId(331, nameof(JobHostFunctionTimeoutNotSet)),
+                "FunctionTimeout is not set.");
 
         private static readonly Action<ILogger, string, Exception> _autorestGeneratedFunctionApplication =
-            LoggerMessage.Define<string>(
-            LogLevel.Information,
-            new EventId(332, nameof(AutorestGeneratedFunctionApplication)),
-            "autorest_generated.json file found generated by Autorest (https://aka.ms/stencil) | file content:\n{sanitizedAutorestJson}");
+            LoggerMessage.Define<string>(LogLevel.Information,
+                new EventId(332, nameof(AutorestGeneratedFunctionApplication)),
+                "autorest_generated.json file found generated by Autorest (https://aka.ms/stencil) | file content:\n{sanitizedAutorestJson}");
 
         private static readonly Action<ILogger, string, Exception> _incorrectAutorestGeneratedJSONFile =
-            LoggerMessage.Define<string>(
-            LogLevel.Warning,
-            new EventId(333, nameof(IncorrectAutorestGeneratedJsonFile)),
-            "autorest_generated.json file found is incorrect (https://aka.ms/stencil) | exception:\n{contents}");
+            LoggerMessage.Define<string>(LogLevel.Warning,
+                new EventId(333, nameof(IncorrectAutorestGeneratedJsonFile)),
+                "autorest_generated.json file found is incorrect (https://aka.ms/stencil) | exception:\n{contents}");
+
+        private static readonly Action<ILogger, string, bool, bool, bool, Exception> _scriptStartUpNotLoadingExtensionBundle =
+            LoggerMessage.Define<string, bool, bool, bool>(LogLevel.Information,
+                new EventId(334, nameof(ScriptStartNotLoadingExtensionBundle)),
```

Comment:
The event ID was incorrect here (duplicated). This will change logging, but we had a conflict before.